All Core Devs Meetings/Meeting 81.md

```diff
@@ -34,18 +34,18 @@ EIP-1985 | Discussed under `EFI`. Decision required around needing a Hard Fork
 
 **DECISIONS 81.4**: There is no way can to confirm the EIP 1962 is going in, clients have serious concerns implementing. They may coordinate something in ETH Paris. 
 
-**DECISIONS 81.5**: Berlin would happen when the BLS pre-compile is ready, ProgPOW would happen the next third Wednesday after that.
-
 
 ## ACTION ITEMS
 
-**ACTION 81.1**: Ethereum blog to announce officially that ProgPOW is being adopted because of the decision of all core devs.
+**ACTION 81.1**: Zach will share better communication methods to interact with the Open RPC team. 
 
-**ACTION 81.2**: Zach will share better communication methods to interact with the Open RPC team. 
+**ACTION 81.2**: Hudson will merge PR to change the state to Accepted for EIP 778 and EIP 868. 
 
-**ACTION 81.3**: Hudson will merge PR to change the state to Accepted for EIP 778 and EIP 868. 
+## PROPOSALS
 
+**PROPOSAL 81.1**: Berlin would happen when the BLS pre-compile is ready, ProgPOW would happen the next third Wednesday after that.
```

Comment:
I believe it's the third Wednesday of the following month. Network Upgrades are targetted for 3rd Wednesdays of the month they are assigned.

Reply:
Just making sure people don't interpret it as 3 weeks later, instead of a full month later.

Reply:
In the call, the discussion was about 3 weeks. Ref. line no. 434.

Reply:
Oh, they were discussing as if it were 3 weeks. 3rd Wednesday in the new EIP centric process means 3rd Wednesday of the month. Or at least that's what Danno and Tim were proposing. https://www.youtube.com/watch?v=vsA57h052hU